libcloudforensics/providers/aws/forensics.py

```diff
@@ -239,8 +243,11 @@ def StartAnalysisVm(
         will not be accessible. It is therefore recommended to fill in this
         parameter.
     tags (Dict[str, str]): Optional. A dictionary of tags to add to the
-          instance, for example {'TicketID': 'xxx'}. An entry for the instance
-          name is added by default.
+        instance, for example {'TicketID': 'xxx'}. An entry for the instance
+        name is added by default.
+    subnet_id (str): Optional. The subnet to launch the instance in
```

Comment:
Super-nit for this and elsewhere in the changes you made: add a `.` at the end of the sentences. This helps making our auto-generated documentation on ReadTheDocs more readable :).

Reply:
I think I got em all!